ATV:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  

  # Position Controllers --------------------------------------- (used to change orientation of front wheels)  = STEERING 
  FR_wheel_position_controller:
    type: effort_controllers/JointPositionController
    joint: right_steering_joint
    pid: {p: 1000.0, i: 100.0, d: 10.0}

  FL_wheel_position_controller:
    type: effort_controllers/JointPositionController
    joint: left_steering_joint
    pid: {p: 1000.0, i: 100.0, d: 10.0}

 # Velocity Controllers --------------------------------------- (used for forward/backword rotation of rear wheels)  =  MOVING
  RR_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: right_rear_axle
    pid: {p: 800.0, i: 200.0, d: 0.1}

  RL_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: left_rear_axle
    pid: {p: 800.0, i: 200.0, d: 0.1}